tests/test_memnn.py

```diff
@@ -77,6 +77,39 @@ def test_labelcands_multi(self):
             "test hits@1 = {}\nLOG:\n{}".format(test['hits@1'], stdout),
         )
 
+    def test_backcomp(self):
+        '''
+        Tests that the memnn model files continue to works over time
+        '''
+        testing_utils.download_unittest_models()
+
+        stdout, valid, test = testing_utils.eval_model(
+            dict(
+                task='integration_tests',
+                model='memnn',
+                model_file='zoo:unittest/memnn/model',
+                dict_file='zoo:unittest/memnn/model.dict',
+                batch_size=16,
+            )
+        )
+
+        self.assertGreaterEqual(
+            valid['accuracy'],
+            0.88,
```

Comment:
was hoping for a higher number but alright.

Reply:
I could try re-training the model with a different learning rate/other parameters or for a longer time if necessary. Also Kurt has already uploaded the model.

Reply:
dope, this is perfect then.